src/games/cards.ts

```diff
@@ -0,0 +1,520 @@
+import { ChatColor } from 'net.md_5.bungee.api';
+import { Material, SoundCategory } from 'org.bukkit';
+import { BlockFace } from 'org.bukkit.block';
+import { Item, Player } from 'org.bukkit.entity';
+import { PlayerInteractEvent } from 'org.bukkit.event.player';
+import { EquipmentSlot, ItemStack } from 'org.bukkit.inventory';
+import { Vector } from 'org.bukkit.util';
+import { CustomItem } from '../common/items/CustomItem';
+import * as yup from 'yup';
+import { Action } from 'org.bukkit.event.block';
+import { isRightClick } from '../common/helpers/click';
+
+const ZERO_VECTOR = new Vector();
+const PICKUP_DELAY = 12000; // TICKS -> 10 minutes
+const MAX_CARDS_IN_DECK = 54; // 52 cards + 2 jokers
+const SHUFFLE_COOLDOWN = 1250; // milliseconds
+
+const CARD_FLIP_SOUND = 'custom.card';
+const DECK_SHUFFLE_SOUND = 'custom.shuffle';
+
+const CARDS = new Map<string, CustomItem<{}>>();
+const SHUFFLE_COOLDOWNS = new Set<Player>();
+
+/*
+ *  Defining deck items
+ */
+const FULL_DECK = new CustomItem({
```

Comment:
```suggestion
const FullDeck = new CustomItem({
```
Custom items have been named as classes. Aside doing it the same way everywhere, it helps distinguish them from constants (e.g. CARDS) and mutable top-level variables (e.g. shuffleCooldows, see my suggestion above).